### ISSUE TITLE
Unable to compose docker for sandbox

### ISSUE THREAD

**Issue by stekhoven** [sib-swiss/dsSwissKnife-example#1]
I tried composing the docker to test the example R script `dsSwissKnife_example.R`, however, I failed at running the docker-compose. 

When issuing: `docker-compose up -d`
I received:
```
    Traceback (most recent call last):
    File "/usr/local/Cellar/docker-compose/1.28.5/libexec/lib/python3.9/site-packages/urllib3/connectionpool.py", line 699, in urlopen
    httplib_response = self._make_request(
    ...
    self.connect()
    File "/usr/local/Cellar/docker-compose/1.28.5/libexec/lib/python3.9/site-packages/docker/transport/unixconn.py", line 43, in connect
    sock.connect(self.unix_socket)
    FileNotFoundError: [Errno 2] No such file or directory
```

**Comment by IulianD** [comment]
Hello,
I'm  not an expert in docker, I found though some similar problems on the net, for instance here:
https://github.com/docker/compose/issues/7896
The suggestion seems to be that the docker process is not running.
Is this a fresh installation of docker? If not, are you able to run any other docker container on the machine?

**Comment by stekhoven** [comment]
The docker deamon was not running, because I installed it using `brew install docker` which does not work, because docker is a system-level package. One needs to use
```
brew install --cask docker
```
Found this information here: https://stackoverflow.com/questions/44084846/cannot-connect-to-the-docker-daemon-on-macos

After this I started docker (Cmd+Space, type `docker` and Enter) - will ask for password to get privileges.

Afterwards, `docker-compose up -d` worked like a charm.
